sdcm/cluster.py

```diff
@@ -4394,9 +4371,11 @@ def _rotate_kms_key(kms_key_alias_name, kms_key_rotation_interval, db_cluster):
                         message=f"Failed to rotate AWS KMS key for the '{kms_key_alias_name}' alias",
                         traceback=traceback.format_exc()).publish()
                 try:
-                    target_node = [node for node in db_cluster.nodes if not node.running_nemesis][0]
                     self.log.debug("Target node for 'rotate_kms_key' is %s", target_node.name)
```

Comment:
The `target_node` is not defined at this step.
The `precommit` also complains about it.

Reply:
moved the line into the context manager